src/scheduler/workers/unpublish/day/index.ts

```diff
@@ -0,0 +1,20 @@
+import { findListsInAnnualReviewForReminders } from "../findListsInAnnualReviewForReminders";
+import { findListsToResetAnnualReview } from "scheduler/workers/unpublish/day/changeState/findListsToResetAnnualReview";
+import { sendUnpublishEmails } from "scheduler/workers/unpublish/day/sendUnpublishEmails";
+import { schedulerLogger } from "scheduler/logger";
+import { changeState } from "scheduler/workers/unpublish/day/changeState";
+
+export async function main() {
+  const logger = schedulerLogger.child({ method: "unpublish day" });
+
+  const listsInAnnualReview = await findListsInAnnualReviewForReminders();
+  const listsToResetAnnualReview = await findListsToResetAnnualReview();
```

Comment:
because `unpublishProvidersAndResetList()` and `sendEmails()` are called once for each List don't we need to settle all the promises in unpublish/day/index.ts?